dotnet/src/Connectors/Connectors.GoogleVertexAI/Extensions/GoogleAIServiceCollectionExtensions.cs

```diff
@@ -137,4 +138,59 @@ public static class GoogleAIServiceCollectionExtensions
                 loggerFactory: serviceProvider.GetService<ILoggerFactory>()));
         return services;
     }
+
+    /// <summary>
+    /// Add Google AI embeddings generation service to the kernel builder.
+    /// </summary>
+    /// <param name="builder">The kernel builder.</param>
+    /// <param name="modelId">The model for text generation.</param>
+    /// <param name="apiKey">The API key for authentication Gemini API.</param>
+    /// <param name="serviceId">The optional service ID.</param>
+    /// <param name="httpClient">The optional custom HttpClient.</param>
+    /// <returns>The updated kernel builder.</returns>
+    public static IKernelBuilder AddGoogleAIEmbeddingsGeneration(
```

Comment:
Current naming for Azure OpenAI is `AddAzureOpenAITextEmbeddingGeneration`

Can you use `AddGoogleAIEmbeddingGeneration` i.e. remove `s` to be consistent. Need to make this change in multiple places.

Reply:
Ok I fix it

Reply:
done